js/scripts.js

```diff
@@ -0,0 +1,123 @@
+var newGameBtn = document.getElementById('js-newGameButton');
+var pickRock = document.getElementById('js-playerPick_rock');
+var pickPaper = document.getElementById('js-playerPick_paper');
+var pickScissors = document.getElementById('js-playerPick_scissors');
+var gameState = 'notStarted';
+var player = {
+    name: '',
+    score: 0
+};
+var computer = {
+    score: 0
+};
+var newGameElem = document.getElementById('js-newGameElement');
+var pickElem = document.getElementById('js-playerPickElement');
+var resultsElem = document.getElementById('js-resultsTableElement');
+var playerPointsElem = document.getElementById('js-playerPoints');
+var playerNameElem = document.getElementById('js-playerName');
+var computerPointsElem = document.getElementById('js-computerPoints');
+var playerPickElem = document.getElementById('js-playerPick');
+var computerPickElem = document.getElementById('js-computerPick');
+var playerResultElem = document.getElementById('js-playerResult');
+var computerResultElem = document.getElementById('js-computerResult');
+
+pickRock.addEventListener('click', function() { playerPick('rock') });
+pickPaper.addEventListener('click', function() { playerPick('paper') });
+pickScissors.addEventListener('click', function() { playerPick('scissors') });
+newGameBtn.addEventListener('click', newGame);
+
+function setGameElements() {
+  switch(gameState) {
+    case 'started':
+        newGameElem.style.display = 'none';
+        pickElem.style.display = 'block';
+        resultsElem.style.display = 'block';
+      break;
+    case 'ended':
+        newGameBtn.innerText = 'Jeszcze raz';
+    case 'notStarted':
+    default:
+        newGameElem.style.display = 'block';
+        pickElem.style.display = 'none';
+        resultsElem.style.display = 'none';
+  }
+}
+
+setGameElements();
+
+function setGamePoints() {
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+
+    checkGameWinner();
+    setGameElements();
+}
+
+function newGame() {
+  player.name = prompt('Graczu, wpisz swoje imię', 'imię gracza');
+  if (player.name) {
+    player.score = computer.score = 0;
+    gameState = 'started';
+    setGameElements();
+
+    playerNameElem.innerHTML = player.name;
+    setGamePoints();
+  }
+
+}
+
+function getComputerPick() {
+    var possiblePicks = ['rock', 'paper', 'scissors'];
+    return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+
+
+function playerPick(playerPick) {
+    var computerPick = getComputerPick();
+    
+    playerPickElem.innerHTML = playerPick;
+    computerPickElem.innerHTML = computerPick;
+
+    checkRoundWinner(playerPick, computerPick);
+    setGamePoints();
+}    
+
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+var winnerIs = 'player';
+    if (winnerIs = 'player'){
```

Comment:
to nie ma prawa działać - ten warunek będzie zawsze spełniony - gracz wygra zawsze 10:0

po co Ci ta zmienna `winnerIs` zapomnij o niej i napisz ten kod z jej pominięciem, sprawdzaj kto wygrał i reaguj